### PR TITLE
(Feature) New color palette for NW account

### PR DESCRIPTION
**Pull request by vbaranov**
Closes https://github.com/poanetwork/metamask-extension/issues/38

- npm `jazzicon` is swaped with npm `rockicon` to support arbitrary colors palette.
- NW colors palette is added

### CODE REVIEW

File: old-ui/lib/icon-factory.js [hunk 1]
```diff
@@ -37,7 +38,8 @@ IconFactory.prototype.generateIdenticonSvg = function (address, diameter) {
 // creates a new identicon
 IconFactory.prototype.generateNewIdenticon = function (address, diameter) {
   var numericRepresentation = jsNumberForAddress(address)
-  var identicon = this.jazzicon(diameter, numericRepresentation)
+  this.jazzicon.setColorsPalette(colors)
```
Comment (fvictorio) [inline old-ui/lib/icon-factory.js]: Woudln't it make more sense to have this `setColorsPalette` in the constructor, called just once? Or is there a reason to set it each time an icon is generated?

File: ui/lib/icon-factory.js [hunk 2]
```diff
@@ -37,6 +38,7 @@ IconFactory.prototype.generateIdenticonSvg = function (address, diameter) {
 // creates a new identicon
 IconFactory.prototype.generateNewIdenticon = function (address, diameter) {
   var numericRepresentation = jsNumberForAddress(address)
+  this.jazzicon.setColorsPalette(colors)
```
Comment (fvictorio) [inline ui/lib/icon-factory.js]: Same comment here.

File: old-ui/lib/icon-factory.js (hunk 1 above)
Comment (vbaranov) [reply old-ui/lib/icon-factory.js]: @fvictorio makes sense, done, tnx

File: ui/lib/icon-factory.js (hunk 2 above)
Comment (vbaranov) [reply ui/lib/icon-factory.js]: Done

**Pull request merged**